Makefile

```diff
@@ -132,7 +132,8 @@ container-name:
 push: .push-$(DOTFILE_IMAGE) push-name
 .push-$(DOTFILE_IMAGE): .container-$(DOTFILE_IMAGE)
 ifeq ($(findstring gcr.io,$(REGISTRY)),gcr.io)
-	@gcloud docker -- push $(IMAGE):$(VERSION)
+	@gcloud auth configure-docker
```

Comment:
At least on 176.0.0, this command is not available on the stable release track.

Suggest: @gcloud beta auth configure-docker